### ISSUE TITLE
removeEventListerner is expecting 2 arguments but received 1

### ISSUE THREAD

**Issue by LuminaJJ** [dsalvagni/react-profile-picture#10]
Hi its as the Topic says in react, I think it might be caused by the Javascript value that I am using, but when I unmount the component containing the ProfilePicture component, the removeEventLister throws a Javascript error which crushes my application, could you please help

**Comment by dsalvagni** [comment]
@LuminaJJ are you using the most recent version? It was supposed to be fixed.

**Comment by LuminaJJ** [comment]
Hi @dsalvagni, yes I am, still getting the error no matter how many times I remove it from my packages and add it again

**Comment by LuminaJJ** [comment]
@dsalvagni, also just an observation in your code you say componentWillUnount instead of componentWillUnmount, missing a m, maybe that could be the cause

**Comment by dsalvagni** [comment]
@LuminaJJ  thanks for letting me know. Could you please share the error stack?  I fixed that typo btw.

**Comment by tumenor** [comment]
I'm experiencing this issue too.

```
index.js:1 Uncaught TypeError: Failed to execute 'removeEventListener' on 'EventTarget': 2 arguments required, but only 1 present.
    at index.js:1
    at Array.forEach (<anonymous>)
    at t.value (index.js:1)
    at callComponentWillUnmountWithTimer (react-dom.development.js:21896)
    at HTMLUnknownElement.callCallback (react-dom.development.js:336)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:385)
    at invokeGuardedCallback (react-dom.development.js:440)
    at safelyCallComponentWillUnmount (react-dom.development.js:21903)
    at commitUnmount (react-dom.development.js:22392)
    at commitNestedUnmounts (react-dom.development.js:22486)
    at unmountHostComponents (react-dom.development.js:22810)
    at commitDeletion (react-dom.development.js:22896)
    at commitMutationEffects (react-dom.development.js:25323)
    at HTMLUnknownElement.callCallback (react-dom.development.js:336)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:385)
    at invokeGuardedCallback (react-dom.development.js:440)
    at commitRootImpl (react-dom.development.js:25050)
    at unstable_runWithPriority (scheduler.development.js:697)
    at runWithPriority$2 (react-dom.development.js:12149)
    at commitRoot (react-dom.development.js:24922)
    at finishSyncRender (react-dom.development.js:24329)
    at performSyncWorkOnRoot (react-dom.development.js:24307)
    at react-dom.development.js:12199
    at unstable_runWithPriority (scheduler.development.js:697)
    at runWithPriority$2 (react-dom.development.js:12149)
    at flushSyncCallbackQueueImpl (react-dom.development.js:12194)
    at flushSyncCallbackQueue (react-dom.development.js:12182)
    at discreteUpdates$1 (react-dom.development.js:24423)
    at discreteUpdates (react-dom.development.js:1438)
    at dispatchDiscreteEvent (react-dom.development.js:5881)
```

**Comment by tumenor** [comment]
Sadly I am not that familiar with how pull requests work, but I was able to solve this issue.
Reading the documentation at Mozilla, it's evident that the removeEventListener is wrong.

This is how you added the event:
`window.addEventListener(
        eName,
        this.onMovingEnd,
        false
      );`

And this is how you removed it:
`window.removeEventListener(eName, this.onMoving)`


To solve this issue it should be removed in this way:
`window.removeEventListener(eName, this.onMoving, false)`

I've already tested this on my platform and it seems to be working perfect.
I would do a pull request, but I don't know how. :)

Love how easy this project is to use though!
